src/main/java/org/example/homework5/NumberGenerator.java

```diff
@@ -0,0 +1,41 @@
+package org.example.homework5;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class NumberGenerator {
+    public static void main(String[] args) {
+        List<Integer> randomArray = generateRandomArray();
+        arraySum(randomArray);
+    }
+    public static List<Integer> generateRandomArray() {
+        List<Integer> array = new ArrayList<Integer>();
+        boolean flag = true;
+
+        while (flag) {
+            Random random = new Random();
+            int randomNumber = random.nextInt(100) + 1;
+
+            if (array.indexOf(randomNumber) >= 0) {
+                flag = false;
```

Comment:
```suggestion
                break;
```
If you use break you don't need redundant flag variable, and readability looks better

Reply:
Fixed